python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -946,12 +958,32 @@ def _make_request(
         raise Failure(f"Max retries ({self.request_max_retries}) exceeded with url: {url}.")
 
     def get_connections(self) -> Mapping[str, Any]:
-        """Fetches all connections of an Airbyte workspace from the Airbyte API."""
-        raise NotImplementedError()
+        """Fetches all connections of an Airbyte workspace from the Airbyte REST API."""
+        return self._make_request(
+            method="GET",
+            endpoint="connections",
+            base_url=self.rest_api_base_url,
+            params={"workspaceIds": self.workspace_id},
+        )
+
+    def get_connection_details(self, connection_id) -> Mapping[str, Any]:
+        """Fetches details about a given connection from the Airbyte Configuration API."""
+        # Using the Airbyte Configuration API to get the connection details, including streams and their configs.
+        # https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/connections/get
```

Comment:
this link looks a bit insane. Is this actually right?

Reply:
It is. I could also link the [config.yaml](https://github.com/airbytehq/airbyte-platform/blob/v1.1.0/airbyte-api/server-api/src/main/openapi/config.yaml) file from their repo - it's just harder to browse.

Reply:
I can link both though